packages/seacas/libraries/suplib/convert.C

```diff
@@ -1,19 +0,0 @@
-#include <fmt/format.h>
-
-extern "C" {
-#if defined(ADDC_)
-void convert_(char *str, int64_t *value, int64_t *length, int len)
```

Comment:
If you want to try this again at some point, best practice would be to use Fortran's `ISO_C_BINDING` to declare the function to have C binding.  That way, you wouldn't need to worry about underscores or how Fortran passes strings (some Fortran ABIs pass strings as a pointer and a length, etc.).

Reply:
Yes, at some point all of the C/Fortran interactions should be converted to `ISO_C_BINDING`, but the priority never makes it to the top...

The main issue with this change was use of C++ in a Fortran application...